sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/common/InstrumentType.java

```diff
@@ -10,7 +10,7 @@
   COUNTER,
   UP_DOWN_COUNTER,
   HISTOGRAM,
-  OBSERVABLE_SUM,
```

Comment:
as this is a public API, used for views, I think we should probably go through a deprecation cycle for these values.

Reply:
interesting that this doesn't show up in the api diffs. 🤔

Reply:
Does the metrics SDK participate in API diffs? I'm not sure if it does.

+1 to deprecation cycle.

Reply:
oh right..it doesn't because it's not stable yet. 🤦🏽